src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/AbstractGroupBatchAdapterRouter.kt

```diff
@@ -40,18 +43,20 @@ abstract class AbstractGroupBatchAdapterRouter<T> : MessageRouter<T> {
     }
 
     override fun subscribe(callback: MessageListener<T>, vararg attributes: String): SubscriberMonitor? {
-        return groupBatchRouter.subscribe({ consumerTag: String, message: MessageGroupBatch ->
-                callback.handle(consumerTag, buildFromGroupBatch(message))
+        val listener = ConfirmationListener.wrap(callback)
+        return groupBatchRouter.subscribeWithManualAck({ consumerTag: DeliveryMetadata, message: MessageGroupBatch, confirmation ->
+                listener.handle(consumerTag, buildFromGroupBatch(message), confirmation)
```

Comment:
in this case we make an instance of MCL, but don't call the conformation method.